### ISSUE TITLE
PHP debug panel not opening

### ISSUE THREAD

**Issue by shemayas** [gwomacks/php-debug#206]
Hi,
I'm using the plugin for few months and suddenly the panel doesn't open.

![peek 2016-12-27 15-50](https://cloud.githubusercontent.com/assets/11064714/21500966/a3e6ba6e-cc4c-11e6-8d1a-5a143b1e1b40.gif)

Thank you very much.

**Comment by simondelamarre** [comment]
+1

**Comment by simondelamarre** [comment]
i'm using Atom 1.12.9 with php debug plugin version 0.0.1.
i have the same problem sometime its work sometime the panel do not display.
after desinstall reinstall the problem come back.
this is my config :
```
PathMaps: [
      "remotepath;localpath"
      "/var/www/;/Users/sdelamarre/Sites/"
    ]
    ServerPort: 9999
    currentConsoleHeight: "171px"
    currentPanelHeight: "0px"
    currentPanelMode: "bottom"
```

Need help please ! :-)

**Comment by joestock** [comment]
Hi, I wish I had seen this sooner.  If you haven't figured this one out yet, the problem is this:
When you drag the height of the debugger window down to the bottom it sets currentPanelHeitght: "0px" in some cases.  in your config file change it to currentPanelHeight: "40px". save it and then close and restart atom. That should fix it for ya.

**Comment by cchamplin** [comment]
This issue should be resolved in 0.2.5. The panel should no longer allow itself to become 0px tall.
